packages/mock/src/client.ts

```diff
@@ -142,6 +142,13 @@ export class MockClient extends MedplumClient {
     return super.getActiveLogin();
   }
 
+  getLogins(): LoginState[] {
+    if (this.activeLoginOverride !== undefined) {
```

Comment:
What's the uses for this change for the CLI update?

Reply:
This is an oddity of `MockClient`.  By default, `MockClient` is logged in a "Dr. Alice Smith", which is a default `Practitioner` resource that you can use in tests.  `activeLoginOverride` is a `MockClient` feature to modify the current mocked login state.